torch_brain/utils/callbacks.py

```diff
@@ -20,7 +20,7 @@ def on_train_epoch_start(self, trainer, pl_module):
     def on_train_epoch_end(self, trainer, pl_module):
         if self.enable:
             epoch_time = time.time() - self.start_time
-            pl_module.log("epoch_time", epoch_time)
+            pl_module.log("epoch_time", epoch_time, sync_dist=True)
```

Comment:
why do we need to sync_dist here? isn't everything blocked until the last gpu is done? also, why does this need to be accurate?

Reply:
It's not needed in theory. Lightning complains if we don't have sync_dist=True. I'd rather not see that warning. This incurs negligible cost.